### PR TITLE
Proposed fix to #278

### PR DESCRIPTION
**Pull request by gugahoa**
#278 
Add isCheck variable to onEquip to differentiate between when it's being called from queryAdd and when it's being called from postAdd.
It's added to onDeEquip too, but since it's always called from postRemove, isCheck will always be false

### CODE REVIEW

File: src/movement.cpp [hunk 1]
```diff
@@ -820,16 +820,16 @@ bool MoveEvent::executeStep(Creature* creature, Item* item, const Position& pos)
 uint32_t MoveEvent::fireEquip(Player* player, Item* item, slots_t slot, bool boolean)
 {
 	if (scripted) {
-		return executeEquip(player, item, slot);
+		return executeEquip(player, item, slot, boolean);
```
Comment (djarek) [inline src/movement.cpp]: While you're at it, you could change the name of the boolean variable to something that makes more sense.

Comment (gugahoa) [inline src/movement.cpp]: Done 😃

File: src/movement.cpp [hunk 2]
```diff
@@ -817,19 +817,19 @@ bool MoveEvent::executeStep(Creature* creature, Item* item, const Position& pos,
 	return scriptInterface->callFunction(4, true);
 }
 
-uint32_t MoveEvent::fireEquip(Player* player, Item* item, slots_t slot, bool boolean)
+uint32_t MoveEvent::fireEquip(Player* player, Item* item, slots_t slot, bool isCheck)
```
Comment (ninjalulz) [inline src/movement.cpp]: While you're at it, can't you rename the variable to something more meaningful? Perhaps something that contains the keyword 'abilities'. 🤔

Comment (gugahoa) [inline src/movement.cpp]: Sorry, I can't think of anything more meaningful than isCheck to a boolean that is true when the function is called only to check the return value :(

Comment (ninjalulz) [inline src/movement.cpp]: Okay, haha 😄

**Comment by gugahoa** [comment]
Review status? @djarek \cc @ranisalt

**Comment by djarek** [comment]
Thanks!

**Pull request merged**